test/org/traccar/protocol/WatchProtocolDecoderTest.java

```diff
@@ -101,7 +111,40 @@ public void testDecode() throws Exception {
 
         verifyPosition(decoder, buffer(
                 "[ZJ*014111001350304*0038*008a*UD,070318,021027,V,00.000000,N,000.000000,E,0,0,0,0,100,18,1000,50,00000000,4,255,460,0,9346,5223,42,9346,5214,20,9784,4083,11,9346,5221,5]"));
+    }
+
+    @Test
+    public void testDecodeTK() throws Exception {
```

Comment:
Wrong naming convention. Also probably should have better name.

Reply:
Please when you write comments about names give some suggestions. I don't see what's the problem with this name, it explains exactly what it does.

Reply:
Name should be camel-case. Name should be more obvious what it does. I don't know what TK means, so I don't really know what it tests.